<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>178</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Duracion:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="duracion">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>280</y>
      <width>149</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>320</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Distribucion normal de estadia:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Distribucion uniforme de Aterrizajes:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>410</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Distribucion exponencial negativa de llegadas:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Distribucion uniforme de Despegues:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>470</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Capacidad Maxima:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="bAterrizaje">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>339</y>
      <width>139</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="dsvStd">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>308</y>
      <width>139</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="bDespegue">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>370</y>
      <width>139</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="capMax">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>470</y>
      <width>139</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>520</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>180</width>
      <height>133</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Tiempo inicial de llegada: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Finalizacion de permanencia 1:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Finalizacion de permanencia 2:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Finalizacion de permanencia 3:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Finalizacion de permanencia 4:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Finalizacion de permanencia 5:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>310</y>
      <width>141</width>
      <height>119</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLineEdit" name="media"/>
     </item>
     <item>
      <widget class="QLineEdit" name="aAterrizaje"/>
     </item>
     <item>
      <widget class="QLineEdit" name="aDespegue"/>
     </item>
     <item>
      <widget class="QLineEdit" name="mediaExpo"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>161</width>
      <height>177</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="tiempoPrimeraLlegada"/>
       </item>
       <item>
        <widget class="QLineEdit" name="permanencia1"/>
       </item>
       <item>
        <widget class="QLineEdit" name="permanencia2"/>
       </item>
       <item>
        <widget class="QLineEdit" name="permanencia3"/>
       </item>
       <item>
        <widget class="QLineEdit" name="permanencia4"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="permanencia5"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
